st_louis/__init__.py

```diff
@@ -0,0 +1,74 @@
+# encoding=utf-8
+from pupa.scrape import Jurisdiction, Organization
+from .people import StLouisPersonScraper
+from .bills import StLouisBillScraper
+
+
+class StLouis(Jurisdiction):
+    division_id = "ocd-division/country:us/state:mo/place:st_louis"
+
+    # FIXME
+    # Do we need jurisdiction_id? pupa did not auto-generate this field 
+    # like division_id, I made it myself.
+    # And is this the correct id? or should it be 
+    # .../place:st_louis/council ? or .../place:st_louis/government ?
+    jurisdiction_id = "ocd-jurisdiction/country:us/state:mo/place:st_louis"
```

Comment:
You shouldn't need to do this. just declare division id and classification like you do.